storage/innobase/dict/dict0stats.cc

```diff
@@ -2877,11 +2998,26 @@ dict_stats_report_error(dict_table_t* table, bool defragment)
 	return err;
 }
 
-/** Save the persistent statistics of a table or an index.
-@param table            table whose stats to save
-@param only_for_index   the index ID to save statistics for (0=all)
-@return DB_SUCCESS or error code */
-dberr_t dict_stats_save(dict_table_t* table, index_id_t index_id)
+/** Return a display name for the innodb_stats_method
+@param  stat_method   innodb_stats_method during index statistics
+@return innodb_stats_method name */
+static const char* get_innodb_stats_method(unsigned stat_method)
+{
+  switch (stat_method) {
+  case SRV_STATS_NULLS_IGNORED:
+    return " NULLS_IGNORED";
```

Comment:
Is there a particular reason why we do not use a table here?
```c++
static const char *stats_method_name[]= {
  nullptr, " NULLS_UNEQUAL", " NULLS_IGNORED"
};
static const char *get_innodb_stats_method(unsigned stat_method)
{
  ut_ad(stat_method < 3);
  static_assert(SRV_STATS_NULLS_EQUAL == 0, "");
  static_assert(SRV_STATS_NULLS_UNEQUAL == 1, "");
  static_assert(SRV_STATS_NULLS_IGNORED == 2, "");
  return stat_method < 3 ? stat_method_names[stat_method] : nullptr;
}
```
Also, the value `nullptr` seems to require an additional condition in the caller. Can we just return `""` instead of that?